go/porcelain/deploy.go

```diff
@@ -95,6 +95,7 @@ type DeployOptions struct {
 	files             *deployFiles
 	functions         *deployFiles
 	functionSchedules []*models.FunctionSchedule
+	functionsConfig   map[string]models.FunctionConfig
```

Comment:
based on the previous properties, files, functions and functionSchedules are not required options, but if I am not wrong, now functionsConfig will be required to be at least an empty map?

Reply:
It was not meant as being required. But are you saying this because of the usage of `*` that the previous ones were considered not required? :)

Reply:
🤔 yeah it's really new for me as well. I thought you could make it work like this as well because the map should not throw an error if it's not available. And I think in the swagger file, it was not set up as being required, so I'm thinking maybe it will use that as well to know wether or not the field itself should be required or not. But maybe @eduardoboucas can shine some light on this as well :)